src/guide/components/v-model.md

```diff
@@ -19,7 +19,7 @@ function update() {
 </script>
 
 <template>
-  <div>Parent bound v-model is: {{ model }}</div>
+  <div @click="update">Parent bound v-model is: {{ model }}</div>
```

Comment:
**suggestion**: I agree with the addition, but let's make it a bit more semantic by adding a button and binding the `update` to it:

```suggestion
  <div>Parent bound v-model is: {{ model }}</div>
  <button @click="update">Increment</button>
```

Reply:
Yeah, that makes total sense :)

Reply:
Thank you for the speedy fix! :)